Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -298,3 +291,21 @@ extension Module  {
         return buildSettings
     }
 }
+
+extension Plist {
+    // FIXME: This is only internal for unit testing.
+    /// Quotes the strings in the Plist.
+    func quoteStrings() -> Plist {
+        switch self {
+        case .string(let str):
+            return .string("\"" + str + "\"")
```

Comment:
won't that be taken care when serialize() is called? or you mean it should be escaped twice?